### PR TITLE
Homepage components

### PR DESCRIPTION
**Pull request by rishabhborn2win**
This is the book component which contains about the website and about the developer:
book

This is the component describing how can you help using this website:
cause

This is the review section component:
review

This is the top Stories carousel:
stories

Added all the SS in the PR.

### CODE REVIEW

**Comment by rishabhborn2win** [comment]
This is the updated what do people think section!

image

File: client/public/index.html [hunk 1]
```diff
@@ -25,6 +25,15 @@
...
     <title>React App</title>
+    <!-- added bootstrap and jquery library --rishabhbor2win -->
+    <link href="//netdna.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+<script src="//netdna.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
+    <link href="//netdna.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx" crossorigin="anonymous"></script>
+
+<!-- Closed Addition -->
```
Comment (divyaprakash-Rx) [inline client/public/index.html:36]: @rishabhborn2win please don't edit index.html or add cdns to it. Use npm modules instead.

**Review by divyaprakash-Rx — CHANGES_REQUESTED** [review]
@rishabhborn2win Everything seems fine and nice.
Please run 'npm run format' on the code sync the fork with main repo.

**Review by divyaprakash-Rx — CHANGES_REQUESTED** [review]
@rishabhborn2win Everything is fine now.
Just three things:-
- You have one unnecessary package-lock.json in root folder(the folder of which client and server are subfolders), please delete that.
- Transfer all the components file of homepage to src/components/home from src/components.
- Use prettier before merging.

**Comment by rishabhborn2win** [comment]
@divyaprakash-Rx Have done all the updates which you told.

**Review by divyaprakash-Rx — APPROVED** [review]

**Review by divyaprakash-Rx — APPROVED** [review]

**Comment by rishabhborn2win** [comment]
The styles have got some errors when it got merged with the master branch.

**Comment by divyaprakash-Rx** [comment]
> 
> 
> The styles have got some errors when it got merged with the master branch.

It happens because some are overwriting the default css classes of libraries.

**Pull request closed without merging**